app/main.py

```diff
@@ -1,3 +1,48 @@
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, x, y):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        if isinstance(other, Vector):
+            return Vector(self.x + other.x, self.y + other.y)
+
+    def __sub__(self, other):
+        if isinstance(other, Vector):
+            return Vector(self.x - other.x, self.y - other.y)
+
+    def __mul__(self, other):
+        if isinstance(other, (int, float)):
+            return Vector(self.x * other, self.y * other)
+        if isinstance(other, Vector):
+            return self.x * other.x + self.y * other.y
+
+    @classmethod
+    def create_vector_by_two_points(cls, start_point, end_point):
+        return cls(end_point[0] - start_point[0],
+                   end_point[1] - start_point[1])
+
+    def get_length(self):
+        return (self.x * self.x + self.y * self.y) ** 0.5
+
+    def get_normalized(self):
+        return Vector(self.x / self.get_length(), self.y / self.get_length())
```

Comment:
you twice called self.get_length(), it's better to save result of self.get_length() and use it